.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature-request
+assignees: ''
+
+---
+
+### Current situation
+
+A clear and concise description of what the problem is. Ex. I'm always frustrated when
```

Comment:
Typical feature request will be that some feature from the tax/transfer system is missing. Please use that as the example.